interpreter/runtime/table.mli

```diff
@@ -11,11 +11,12 @@ exception Type
 exception Bounds
 exception SizeOverflow
 exception SizeLimit
+exception OutOfMemory
 
-val alloc : table_type -> table
+val alloc : table_type -> ref_ -> table
```

Comment:
Nit: add comment that this can raise OutOfMemory?

Reply:
Done.